### PR TITLE
Allowing missing file paths to be ignored

### PR DESCRIPTION
**Pull request by JeffAshton**

### CODE REVIEW

File: src/LaunchDarkly.ServerSdk/Files/FileDataSource.cs [hunk 1]
```diff
@@ -88,6 +90,10 @@ private void LoadAll()
                     var data = _parser.Parse(content);
                     data.AddToData(allData);
                 }
+                catch (FileNotFoundException) when (_skipMissingPaths)
```
Comment (JeffAshton) [inline src/LaunchDarkly.ServerSdk/Files/FileDataSource.cs]: To enable the scenario I described yesterday:


Watching current.json and old.json

```
File.Move( "current.json", "old.json" );  // old would get reloaded with same rules
File.Move( "new.json", "current.json" );  // merge of old and new would get reloaded
File.Delete( "old.json" );  // new would get reloaded
```

**Comment by eli-darkly** [comment]
Looks good, thanks.

**Comment by eli-darkly** [comment]
A unit test would be good, though. I'll add one.

**Comment by eli-darkly** [comment]
Released this change in 5.9.0.

**Pull request merged**